<ui version="4.0">
 <class>HostMenu</class>
 <widget class="QWidget" name="HostMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="mapSelectButton">
     <property name="text">
      <string>Select map..</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QPushButton" name="startGameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QListWidget" name="clientListWidget"/>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QPushButton" name="hostButton">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Players:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="playerNameLabel">
       <property name="text">
        <string>Player Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="playerNameLineEdit">
       <property name="text">
        <string>Unnamed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="portLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="maxScore">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Max score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="livesSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="serverIPLabel">
     <property name="text">
      <string>Server Ip(s)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTextEdit" name="serverIpsTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>HostMenu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
